src/azure-cli/azure/cli/command_modules/acs/_help.py

```diff
@@ -479,6 +479,24 @@
   - name: --enable-keda
     type: bool
     short-summary: Enable KEDA workload auto-scaler.
+  - name: --enable-azuremonitormetrics
+    type: bool
+    short-summary: Enable Azure Monitor Metrics Profile
+  - name: --azure-monitor-workspace-resource-id
+    type: string
+    short-summary: Resource ID of the Azure Monitor Workspace
```

Comment:
Could we consider making the naming of parameters clearer and more concise?
```suggestion
  - name: --enable-monitor-metrics
    type: bool
    short-summary: Enable Azure Monitor Metrics Profile
  - name: --monitor-workspace-resource-id
    type: string
    short-summary: Resource ID of the Azure Monitor Workspace
```

Reply:
Hi, We went through the naming process with the AKS team during the LRB Review and finalized on these i.e. they where approved in the [design doc](https://dev.azure.com/msazure/CloudNativeCompute/_wiki/wikis/CloudNativeCompute.wiki/400808/Managed-Prometheus-Addon). Updating it now will require reapprovals and personally I think the azure monnitor metrics name is more clearer and aligned with the broader future strategy for the addon as we will be moving the container insights addon to v2 under a similar name like azuremonitorlogs in the near future.

Do you think its okay if we keep it like this?

Reply:
Okay, I understand. However, it is recommended to change the name of `--enable-azuremonitormetrics` to `--enable-azure-monitor-metrics` to better comply with the naming conventions of CLI

Reply:
updated